clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/validators/StandardGceAttributeValidator.groovy

```diff
@@ -322,15 +300,19 @@ class StandardGceAttributeValidator {
     }
 
     if (location) {
-      def knownLocation = location in vCpuMaxByLocation
+      def knownLocation = location in credentials.locationToInstanceTypesMap
 
       // since our list might be out of date, we'll assume a location can handle at least 16 vCpus per machine.
       if (!knownLocation && vCpuCount > 16) {
```

Comment:
Should this logic still exist? Or is the `locationToInstanceTypesMap` not definitive?